g3doc/projects/vip_deeplab.md

```diff
@@ -38,4 +57,20 @@ results, please use the following BibTeX entry.
 ```
 
 ### References
-Add some related works if any
```

Comment:
Also is there any plan to show some results(numbers and ckpts) as Huiyu did for max-deeplab?

Reply:
I have very limited computation resources right now. I'll try to train some models for them.